docs/angular/guides/configuration.md

```diff
@@ -358,3 +359,7 @@ The syntax is the same as a [`.gitignore` file](https://git-scm.com/book/en/v2/G
 
 1. Changes to that file will not be taken into account in the `affected` calculations.
 2. Even if the file is outside an app or library, `nx workspace-lint` will not warn about it.
+
+## End-to-end Cloud Encryption
```

Comment:
This duplicates a lot of the text in the section above.  I would remove this section and add a not to the command line option saying "You may also provide this value as an environment variable with the key: `NX_CLOUD_ENCRYPTION_KEY`"

Reply:
Moved - good call